app/controllers/concerns/metadata.rb

```diff
@@ -13,7 +13,7 @@ def set_headers
     end
 
     def metadata(opts = {})
-      opts[:total] ||= policy_scope(resource).count
+      opts[:total] ||= scope_search.count
```

Comment:
Notice the definition of this method doesn't include the search if it's not in the request